docs/automated-tests/test-execution/quick-run-baseurl.md

```diff
@@ -1,29 +1,32 @@
 ---
-title: BaseURL Quick Access
-page_title: BaseURL Quick Access
-description: "Test Studio Quick test Execution. Test Studio test run. Quick access the BaseURL. Quick Set BaseURL on project level. "
+title: BaseUrl Quick Access
+page_title: BaseUrl Quick Access
+description: "Test Studio Quick test Execution. Test Studio test run. Quick access the BaseUrl. Quick Set BaseUrl on project level. "
 position: 3
 ---
-# Quick Set BaseURL for a Project
+# Quick Access to BaseUrl
 
-The **Test Ribbon** in Test Studio allows you to trigger a quick test execution. Along with that it provides quick access to some useful project settings.
+The **Test** ribbon in Test Studio allows you to trigger a quick test execution and provides access to the **BaseUrl** project setting.
 
-## Using BaseURL in Test Studio Projects
+## Using BaseUrl in Test Studio Projects
 
-Test Studio allows you to reuse already recorded tests against multiple environments, e.g. stage, test or prod. Usually these environments only differ by their URLs, so setting a BaseURL on project level will help you execute the same tests. The **Test Ribbon** provides quick access to easily change the BaseURL and trigger a test execution against any of the available environments.
+Test Studio allows you to reuse already recorded tests against multiple environments, for example, stage, test, or production. Usually, these environments differ only in their URLs, so changing the **BaseUrl** on the project level will help you execute the same tests against multiple environments.
 
-## Set BaseURL for a Project
+To change the **BaseUrl**, go to the **Test** ribbon and enter the desired URL in the designated field.
 
-Set a **Base Url**, if you run tests against multiple environments. Please see our <a href="/knowledge-base/test-execution-kb/base-url" target="_blank">BaseURL KB article</a> for more information.
```

Comment:
- Don't use _Please_ in docs.
- Use descriptive words in the anchor text for links, see https://testdocs.telerik.com/style-guide/cross-references#guidelines